updatecli/updatecli.d/update-plugins.yaml

```diff
@@ -0,0 +1,143 @@
+    kind: githubrelease
+    spec:
+      owner: "jenkinsci"
+      repository: "jenkins"
+      token: "{{ requiredEnv .github.token }}"
+      versionfilter:
+        kind: regex
+        pattern: 'jenkins-2.303.\d+$'
+    transformers:
+      - trimPrefix: "jenkins-"
+  jenkins2.319.x:
+    name: Get Last Jenkins Stable Version
+    kind: githubrelease
+    spec:
+      owner: "jenkinsci"
+      repository: "jenkins"
+      token: "{{ requiredEnv .github.token }}"
+      versionfilter:
+        kind: regex
+        pattern: 'jenkins-2.319.\d+$'
+    transformers:
+      - trimPrefix: "jenkins-"
+  jenkins2.332.x:
+    name: Get Last Jenkins Stable Version
+    kind: githubrelease
+    spec:
+      owner: "jenkinsci"
+      repository: "jenkins"
+      token: "{{ requiredEnv .github.token }}"
+      versionfilter:
+        kind: regex
+        pattern: 'jenkins-2.332.\d+$'
+    transformers:
+      - trimPrefix: "jenkins-"
+conditions:
+  jenkins2.289.x:
+    name: Test if Jenkins stable published
+    kind: maven
+    sourceid: jenkins2.289.x
+    spec:
+      url: "repo.jenkins-ci.org"
+      repository: "releases"
+      groupid: "org.jenkins-ci.main"
+      artifactid: "jenkins-war"
+  jenkins2.303.x:
+    name: Test if Jenkins stable published
+    kind: maven
+    sourceid: jenkins2.303.x
+    spec:
+      url: "repo.jenkins-ci.org"
+      repository: "releases"
+      groupid: "org.jenkins-ci.main"
+      artifactid: "jenkins-war"
+  jenkins2.319.x:
+    name: Test if Jenkins stable published
+    kind: maven
+    sourceid: jenkins2.319.x
+    spec:
+      url: "repo.jenkins-ci.org"
+      repository: "releases"
+      groupid: "org.jenkins-ci.main"
+      artifactid: "jenkins-war"
+  jenkins2.332.x:
+    name: Test if Jenkins stable published
+    kind: maven
+    sourceid: jenkins2.332.x
+    spec:
+      url: "repo.jenkins-ci.org"
+      repository: "releases"
+      groupid: "org.jenkins-ci.main"
+      artifactid: "jenkins-war"
+targets:
+  jenkins2.289.x:
+    name: 'Update 2.289.x plugins'
+    sourceid: jenkins2.289.x
+    kind: shell
+    spec:
+      command: bash ./updatecli/updatecli.d/update-plugins.sh
+    scmid: github
+  jenkins2.303.x:
+    name: 'Update 2.303.x plugins'
+    sourceid: jenkins2.303.x
+    kind: shell
+    spec:
+      command: bash ./updatecli/updatecli.d/update-plugins.sh
+    scmid: github
+  jenkins2.319.x:
+    name: 'Update 2.319.x plugins'
+    sourceid: jenkins2.319.x
+    kind: shell
+    spec:
+      command: bash ./updatecli/updatecli.d/update-plugins.sh
+    scmid: github
+  jenkins2.332.x:
+    name: 'Update 2.332.x plugins'
+    sourceid: jenkins2.332.x
+    kind: shell
+    spec:
+      command: bash ./updatecli/updatecli.d/update-plugins.sh
+    scmid: github
+pullrequests:
+  default:
+    title: 'Bump pinned plugins'
+    kind: github
+    scmid: github
+    targets:
+      - jenkins2.289.x
+      - jenkins2.303.x
+      - jenkins2.319.x
+      - jenkins2.332.x
```

Comment:
we can do one or multiple PRs for each bom.

Does it matter @timja @basil @jglick ?

Reply:
You mean when a given plugin gets updates for multiple LTS lines? I think it is fine for those to be separate PRs, since this is not the normal case. A plugin will only be mentioned in a `bom-2.*.x/pom.xml` when it is overridden there relative to the newer lines, so when Jenkins baselines are updated across more than one LTS bump in a given plugin release, we would only be adding one such entry.